### DISCUSSION TITLE
L930 - Customized effects request

### DISCUSSION THREAD

**Discussion by ltechdnb** [petretiandrea/home-assistant-tapo-p100#403] [Ideas]
Hi,

Firstly love this integration. 
Thanks for adding "Select Effects" so you can select standard effects from these lights.

I have some Customized Effects on the app the I have made are you able to make it so we can access these and set within HA.

Thanks again. Luke

**Comment by alp82** [comment]
I just added this integration to my HA. I'm one step behind you as I can't see any action names "Select Effects".

All I see is on/off/toggle and brightness increase/decrease. I know you're looking for help yourself, but maybe you can help me here as this might be a small mistake on my side.

**Reply by petretiandrea** [reply]
Are you sure you are integrating a L930?

**Reply by alp82** [reply]
Sorry, my bad. It's a L920.

**Reply by alp82** [reply]
It took some time to appear, but now I have the effect control and it works. But the color wheel does not. All colors I chose end up being received as white.
